pkg/kyverno/common/common.go

```diff
@@ -489,16 +489,18 @@ func ApplyPolicyOnResource(policy *v1.ClusterPolicy, resource *unstructured.Unst
 	resPath := fmt.Sprintf("%s/%s/%s", resource.GetNamespace(), resource.GetKind(), resource.GetName())
 	log.Log.V(3).Info("applying policy on resource", "policy", policy.Name, "resource", resPath)
 
+	resourceRaw, _ := resource.MarshalJSON()
+	updated_resource, err := ut.ConvertToUnstructured(resourceRaw)
```

Comment:
This is now inaccurate. The error occurs not when converting to unstructured. Therefor the error message is also wrong. Error check should always be in the immediately following line.

Reply:
Thanks @MarcelMue. I have updated PR.